setup.py

```diff
@@ -22,7 +22,7 @@
     'copulas>=0.7.0,<0.8',
     'ctgan>=0.5.3.dev0,<0.6',
     'deepecho>=0.3.0.post1,<0.4',
-    'rdt>=1.3.0.dev0,<2.0',
```

Comment:
we can just pin it, otherwise when we do the actual release it'll grab that

Reply:
Sure!